app/src/main/res/layout/content_card_items.xml

```diff
@@ -0,0 +1,22 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+  <data>
+    <variable
+      name="htmlContent"
+      type="String"/>
+  </data>
+  <RelativeLayout
```

Comment:
Can this just be a FrameLayout given that it only has one child?

Reply:
Yes, made it framelayout